fedoracoreos/fcos.go

```diff
@@ -0,0 +1,29 @@
+package fedoracoreos
+
+import "fmt"
+
+const (
+	// StreamStable is the default stream
+	StreamStable = "stable"
+	// StreamTesting is what is intended to land in stable
+	StreamTesting = "testing"
+	// StreamNext usually tracks the next Fedora major version
+	StreamNext = "next"
+)
+
+const (
+	buildHostURL           = "https://builds.coreos.fedoraproject.org/"
+	releaseIndexLocation   = "prod/streams/%s/releases.json"
+	streamMetadataLocation = "streams/%s.json"
+)
+
+// GetStreamURL returns the URL for the given stream
+func GetStreamURL(stream string) string {
+	return buildHostURL + fmt.Sprintf(streamMetadataLocation, stream)
+}
+
+// GetReleaseIndexURL returns the URL for the release index of a given stream.
+// Avoid this unless you have a specific need to test a specific release.
+func GetReleaseIndexURL(stream string) string {
```

Comment:
Since we're currently not providing data structures for the release index, it feels odd to provide a helper for fetching it.

Reply:
I'm trying to shoot now for making cosa(kola) use this and it loads the release index; looking at adding that.